app/assets/javascripts/components/features/compose/containers/sensitive_guide_container.jsx

```diff
@@ -0,0 +1,40 @@
+import { connect } from 'react-redux';
+import TextIconButton from '../components/text_icon_button';
+import { changeComposeSensitivity } from '../../../actions/compose';
+import { Motion, spring } from 'react-motion';
+import { injectIntl, defineMessages } from 'react-intl';
+
+const messages = defineMessages({
+  message: { id: 'compose_form.sensitive_message', defaultMessage: 'Please be sure to mark NSFW if the image you are trying to post is erotic content' }
```

Comment:
"Not safe for work" is a phrase that has come to be a bit of an idiom. When it comes to cross-cultural explanation it may not be clear what that means. A phrasing such as "If the image contains content others may deem inappropriate." or "If an image contains content others may wish not to see"